src/code/z_parameter.c

```diff
@@ -221,27 +221,150 @@ s16 sFinalHoursClockFrameEnvBlue = 0;
 s16 sFinalHoursClockColorTimer = 15;
 s16 sFinalHoursClockColorTargetIndex = 0;
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8010CB80.s")
+Gfx* Gfx_DrawTexRectRGBA16(Gfx* displayListHead, TexturePtr texture, s16 textureWidth, s16 textureHeight, s16 rectLeft,
```

Comment:
Don't we usually just use `gfxP` for gfx pointers?

Reply:
I'm not exactly sure on convention. All the OoT Gfx_ functions in parameter use `displayListHead`, but I can change it to `gfxP` if that's better convention

Reply:
I feel like that's what is used pretty much everywhere else.

Reply:
actually, `gfxP` seems more for double Gfx pointers: `Gfx** gfxP`, so should this just be `(Gfx* gfx, ` for these?

Reply:
Oh right, I think I'm thinking of what's in the prints in OoT, where they use `gfxp` and `gfxpp`.